salt/modules/reg.py

```diff
@@ -18,6 +18,17 @@
 Keys are the folders in the registry. Keys can have many nested subkeys. Keys
 can have a value assigned to them under the (Default)
 
+When passing key on the CLI it must be quoted correctly depending on the
+backslashes being used (``\`` vs ``\\``). The following are valid methods of
+passing the the key on the CLI:
+
+Using single backslashes:
+    ``"SOFTWARE\Python"``
+    ``'SOFTWARE\Python' (will not work on a Windows Master)``
```

Comment:
I think it would make more sense to move the parentheses part outside of the backticks.:
``'SOFWARE\Python'`` (will not work on a Windows Master)

Reply:
Yeah, I'll fix that